content/docs/getting-started/hello-world.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Hello world"
+date: 2019-04-01T10:56:52+01:00
+menu:
+  docs:
+    parent: 'getting-started'
+weight: 101
+---
+
+Let's define an application that's capable of receiving a "hello {name}" message.
+
+```yaml
+asyncapi: '2.0.0'
+id: hello-world-app
+channels:
+  hello:
+    subscribe:
+      message:
+        payload:
+          type: string
+          pattern: '^hello .+$'
+```
+
+Let's get into the details of what we have here:
+
+<pre class="language-yaml line-numbers" data-line="1,2"><code>asyncapi: '2.0.0'
+id: hello-world-app
+channels:
+  hello:
+    subscribe:
+      message:
+        payload:
+          type: string
+          pattern: '^hello .+$'</code></pre>
+
+The first line tells us this is an AsyncAPI document and it's using version 2.0.0 of the specification. This line doesn't have to be the first one but it's a recommended practice.
+
+The second line is an identifier for our application. It is mandatory and must be unique so, when doing real stuff, don't use `hello-world-app` but instead something more "unique" like `urn:com:mycompany:hello-world-app`.
```

Comment:
Replace current text with "The second line identifies the application and is both required and unique. In a real environment using 'urn:com:mycompany:hello-world-app' is considered unique and preferred rather than 'hello-world-app', for example."